docs/tutorials/excel-power-automate-returns.md

```diff
@@ -0,0 +1,183 @@
+            let startDate = convertDate(tableValues[row][2] as number);
+            let endDate = convertDate(tableValues[row][3] as number);
+            if (startDate <= currentDate && endDate >= currentDate) {
+                // Return the first matching email address.
+                return tableValues[row][1].toString();
+            }
+        }
+    }
+
+    // Convert the Excel date to a JavaScript Date object.
+    function convertDate(excelDateValue: number) {
+        let javaScriptDate = new Date(Math.round((excelDateValue - 25569) * 86400 * 1000));
+        return javaScriptDate;
+    }
+    ```
+
+## Create an automated workflow with Power Automate
+
+1. Sign in to the [Power Automate site](https://flow.microsoft.com).
+
+1. In the menu that's displayed on the left side of the screen, press **Create**. This brings you to list of ways to create new workflows.
+
+    ![The Create button in Power Automate](../images/power-automate-tutorial-1.png)
+
+1. Under the **Start from blank** section, select **Scheduled cloud flow**.
+
+    ![The Scheduled cloud flow button in Power Automate](../images/power-automate-return-tutorial-2.png)
+
+1. Now we need to set the schedule for this flow. Our spreadsheet has a new on-call assignment starting every Monday in the first half of 2021. Let's set the flow to run first thing Monday mornings. Use the following options to configure the flow to run on Monday each week:
+
+    - **Flow name**: Notify On-Call Person
+    - **Starting**: 1/4/21 at 1:00am
+    - **Repeat every**: 1 Week
+    - **On these days**: M
+
+    ![A window with the specified options for the scheduled flow](../images/power-automate-return-tutorial-3.png)
+
+1. Press **Create**.
+
+1. Press **New step**.
+
+1. Select the **Standard** tab, then select **Excel Online (Business)**.
+
+    ![The Power Automate option for Excel Online (Business).](../images/power-automate-tutorial-4.png)
+
+1. Under **Actions**, select **Run script (preview)**.
+
+    ![The Power Automate action option for Run script (preview).](../images/power-automate-tutorial-5.png)
+
+1. Next, you'll select the workbook and script to use in the flow step. Use the **on-call-rotation.xlsx** workbook you created in your OneDrive. Specify the following settings for the **Run script** connector:
+
+    - **Location**: OneDrive for Business
+    - **Document Library**: OneDrive
+    - **File**: on-call-rotation.xlsx *(Chosen through the file browser)*
+    - **Script**: Get On-Call Person
+
+    ![The connector settings for running a script in Power Automate.](../images/power-automate-return-tutorial-4.png)
+
+1. Press **New step**.
+
+1. We'll end the flow by sending the reminder email. Select the **Send an email (V2)** by using the connector's search bar. Use the **Add dynamic content** control to add the email address returned by the script. This will be labelled **result** with the Excel icon next to it. You can provide whatever subject and body text you'd like.
```

Comment:
```suggestion
1. We'll end the flow by sending the reminder email. Select **Send an email (V2)** by using the connector's search bar. Use the **Add dynamic content** control to add the email address returned by the script. This will be labelled **result** with the Excel icon next to it. You can provide whatever subject and body text you'd like.
```